src/components/fields/Rating/Icon.tsx

```diff
@@ -0,0 +1,25 @@
+import RatingIcon from "@mui/icons-material/Star";
+import RatingOutlineIcon from "@mui/icons-material/StarBorder"
+import { get } from "lodash-es";
+
+
+export default function Icon(config: any, isEmpty: Boolean) {
```

Comment:
```suggestion
export interface IIconProps {
  config: any;
  isEmpty: boolean;
}

export default function Icon({ config, isEmpty }: IIconProps) {
```
In React components, the first argument is an object of props. You have `config` and `isEmpty` as separate arguments. In this case, `isEmpty` isn’t actually a prop you can pass to the component using `<Icon isEmpty={ ... }>`. So on non-emoji Rating fields, this component always runs `getStateIcon`:
<img width="74" alt="image" src="https://user-images.githubusercontent.com/27017118/198931234-846300e7-552a-483d-a952-fbe83bfee8a8.png">